bincrafters_conventions/bincrafters_conventions.py

```diff
@@ -203,6 +204,7 @@ def _update_compiler_jobs(self, file):
 
     def _update_appveyor_file(self, file):
         update_a_python_version(self, file, python_version_current_appveyor, python_check_for_old_versions)
+        update_python_environment_variable(self, file)
```

Comment:
Please prefix the file name with update_a_ the `a` stands for `appveyor`, `t` for `travis`

Reply:
oh, I was thinking it's just article :)

Reply:
renamed